ctapipe/conftest.py

```diff
@@ -8,6 +8,15 @@
 
 from ctapipe.io import SimTelEventSource
 from ctapipe.utils import get_dataset_path
+from ctapipe.instrument import CameraGeometry
+
+
+@pytest.fixture(scope="session")
+def camera_geometries():
```

Comment:
Is this actually used anywhere?

Reply:
yes, it makes sure we have the required number of known cameras in the test you commented on below.